website/docs/docs/build/measures.md

```diff
@@ -200,16 +200,16 @@ Parameters under the `non_additive_dimension` will specify dimensions that the m
 
 ```yaml
 semantic_models:
-  - name: subscription_id
+  - name: subscription
     description: A subscription table with one row per date for each active user and their subscription plans. 
-    model: ref('your_schema.subscription_table')
+    model: ref('your_schema.subscription')
```

Comment:
You can leave this how it was! The table name doesn't need to match the semantic model name.